docs/src/routes/docs/components/switch.svx

```diff
@@ -0,0 +1,80 @@
+---
+name: Switch
+---
+
+<script>
+  import { Switch } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Switch
+
+An on/off switch.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Switch />
+    <Switch disabled />
+  </div>
+  <div slot="source">
+
+```svelte
+<Switch />
+<Switch disabled />
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`slotLeft`** | `false` | `boolean` | Whether to place the default slot to the left of the switch or to the right. |
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`value`** | `false` | `boolean` | Whether the switch is currently on. |
+| **`disabled`** | `false` | `boolean` | Disables the switch. |
+| **`...`** |  | `any` | Everything else will be passed to the underlying `<input>` element. |
+
+### Class Properties
+
+All `<Switch>` components use the `.switch` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. |
+| **`inputClass`** | `null` | `string` | A class string to add to the `<input>` inside. |
+| **`trackClass`** | `null` | `string` | A class string to add to the [track](https://material.io/components/selection-controls#switches) of the switch. |
+| **`thumbClass`** | `null` | `string` | A class string to add to the [thumb](https://material.io/components/selection-controls#switches) of the switch. |
+
+
+## Events {#events}
+
+| Name | Event Detail | Description |
+| ---- | ------------ | ----------- |
+| **`event`** | `{ value, nativeEvent }` | Fires when a switch is toggled. The `value` corresponds to the current value of the ==`value`== property, the `nativeEvent` contains the [native change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). |
+
+
+## Slots {#slots}
+
+### Default slot
+
+The label of the switch. Positioned to the right of the switch by default, can be moved to the left using the ==`slotLeft`== property.
```

Comment:
Can't help but notice the comma splice here. (yes, I can sometimes be much of a grammar nazi)

Reply:
I had to google what a comma splice is) okay, will fix